pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -354,21 +345,27 @@ func (r *MemberRollReconciler) reconcileObject(ctx context.Context, roll *maistr
 		kialiName := mesh.Status.AppliedSpec.Addons.Kiali.ResourceName()
 		roll.Status.SetAnnotation(statusAnnotationKialiName, kialiName)
 
-		var version versions.Version
-		version, err = versions.ParseVersion(mesh.Spec.Version)
-		if err != nil {
-			return reconcile.Result{}, err
-		}
-		meshIsClusterScoped, err := version.Strategy().IsClusterScoped(&mesh.Spec)
-		if err != nil {
-			return reconcile.Result{}, err
+		if roll.MatchesNamespacesDynamically() {
+			panic("namespace wildcards & selectors for Kiali are not yet implemented")
```

Comment:
panicking doesn't look like a good choice? why not emit a warning and select all namespaces for now?